### ISSUE TITLE
Manually acknowledge tasks

### ISSUE THREAD

**Issue by linar-jether** [celery/celery#3832]
Since setting the **acks_late** or **task_acks_late** for a specific task signature ( i.e. `task_name.s(arg1,arg2).set(queue='test', acks_late=False, task_acks_late=False)`)  
does not seem to work...

Using the redis backend, is it possible to manually acknowledge a task? (Without raising Ignore() as i still want the task to continue running)

I'm asking because I want to disable the acks_late for certain long running tasks so they won't be retried when the visibility_timeout occurs

**Comment by georgepsarakis** [comment]
Did you try with `options`, as shown [here](http://docs.celeryproject.org/en/latest/reference/celery.html#celery.signature):

```
task_name.s(arg1, arg2, options={'acks_late': False})
```

**Comment by linar-jether** [comment]
Yes, this doesn't work... It seems that the worker gets the acks_late option statically from the task class itself, not from the current request

**Comment by georgepsarakis** [comment]
How about using [AsyncResult.revoke](http://docs.celeryproject.org/en/latest/reference/celery.result.html#celery.result.AsyncResult.revoke) with `terminate=False`?

**Comment by ask** [comment]
These are options in the task class only, they're not message options so cannot be specified per invocation sadly.  I guess that is a possible feature request.
